server/db/models/mug.js

```diff
@@ -0,0 +1,38 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Mug = db.define('mug', {
+  name: {
+    type: Sequelize.STRING,
+    unique: true,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  color: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
```

Comment:
take out color for now since my seed file doesnt have it. we can add it later if we want

Reply:
Changed color from allowNull: false => allowNull: true